zeebe/dynamic-config/src/main/java/io/camunda/zeebe/dynamic/config/api/ClusterConfigurationManagementRequest.java

```diff
@@ -42,6 +42,24 @@ public ScaleRequest(final Set<MemberId> members, final boolean dryRun) {
     }
   }
 
+  record ClusterScaleRequest(
```

Comment:
:upside_down_face: It should be possible to rename `ScaleRequest` to the more accurate `BrokerScaleReqest` or something similar. Just to distinguish it from the new and more generic `ClusterScaleRequest`. As far as I remember the name is not serialized anywhere.